packages/postgres-database/src/simcore_postgres_database/models/users_details.py

```diff
@@ -0,0 +1,76 @@
+import sqlalchemy as sa
+
+from ._common import (
+    column_created_datetime,
+    column_modified_datetime,
+    register_modified_datetime_auto_update_trigger,
+)
+from .base import metadata
+from .users import users
+
+users_details = sa.Table(
+    "users_details",
+    #
+    # Provides extra attributes for a user that either not required or that are provided before the user is created.
+    # The latter state is denoted as "pre-registration" and specific attributes in this state are prefixed with `pre_`. Therefore,
+    # a row can be added in this table during pre-registration i.e. even before the `users` row exists.
+    #
```

Comment:
question: why not call it then pre-registryation-details instead?

Reply:
I went through different names and i liked this because it was concrete but yes it might be too generic and will change to your suggetion